contracts/interfaces/ISafe.sol

```diff
@@ -0,0 +1,149 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.7.0 <0.9.0;
+
+import {Enum} from "./IEnum.sol";
+import {IModuleManager} from "./IModuleManager.sol";
+import {IOwnerManager} from "./IOwnerManager.sol";
+import {IFallbackManager} from "./IFallbackManager.sol";
+
+/**
+ * @title ISafe - A multisignature wallet interface with support for confirmations using signed messages based on EIP-712.
+ * @author @safe-global/safe-protocol
+ */
+interface ISafe is IModuleManager, IOwnerManager, IFallbackManager {
```

Comment:
Nice! I was going to say all functions in the interface should be marked as `external`, but you beat me to it! :raised_hands: